src/domain/community/community/community.resolver.mutations.ts

```diff
@@ -189,6 +186,51 @@ export class CommunityResolverMutations {
     return savedApplication;
   }
 
+  @UseGuards(GraphqlGuard)
+  @Mutation(() => ICommunity, {
+    description: 'Join the specified Community as a member.',
```

Comment:
Maybe add something about that an application won't be created to just be explicit.

Reply:
Description is adjusted as per feedback